<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-QVM6J4J" failures="1" tests="10" name="DemoTest" time="0.14" errors="0" timestamp="2023-12-05T09:12:27 IST">
  <testcase classname="TestDemo_2.Test_ng1" name="pqr" time="0.01"/>
  <testcase classname="TestDemo_2.Test_ng1" name="pqr" time="0.002"/>
  <testcase classname="TestDemo_2.Test_ng1" name="pqr" time="0.001"/>
  <testcase classname="TestDemo_2.Test_ng1" name="xyz" time="0.002"/>
  <testcase classname="TestDemo_2.Test_ng1" name="LoginTest" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestDemo_2.Test_ng1.LoginTest(Test_ng1.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LoginTest -->
  <testcase classname="TestDemo_2.Test_ng1" name="HomepageTest" time="0.0">
    <skipped/>
  </testcase> <!-- HomepageTest -->
  <testcase classname="TestDemo_2.Test_ng2" name="m1" time="0.001"/>
  <testcase classname="TestDemo_2.Test_ng2" name="m2" time="0.003"/>
  <testcase classname="TestDemo_2.Test_ng2" name="m3" time="0.001"/>
  <testcase classname="TestDemo_2.Test_ng3" name="p1" time="0.002"/>
  <testcase classname="TestDemo_2.Test_ng1" name="abc" time="0.0">
    <ignored/>
  </testcase> <!-- abc -->
</testsuite> <!-- DemoTest -->
